jobs/release.Jenkinsfile

```diff
@@ -273,11 +273,14 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                 // Run plume to publish official builds; This will handle modifying
                 // object ACLs, modifying AMI image attributes,
                 // and creating/modifying the releases.json metadata index
+                // Note here we pass the bucket instead of `s3_stream_dir` but
+                // plume only actually interacts with objects under the
+                // `s3_stream_dir` key.
```

Comment:
```suggestion
                // Note here we pass the bucket instead of `s3_stream_dir` but
                // plume is hard coded to only interact with objects under the
                // `s3_stream_dir` key (i.e. it doesn't consider `s3.builds_key`).
                // This needs to be revisited in a followup.
```